kratos/sources/cfd_variables.cpp

```diff
@@ -11,16 +11,13 @@
 //
 
 // System includes
-#include <string>
-#include <iostream>
-#include <vector>
 
 // External includes
 
 // Project includes
 #include "includes/define.h"
 #include "includes/cfd_variables.h"
-#include "includes/kernel.h"
```

Comment:
This one is very abused indeed, and it even could go completely off if the variables were registered in the kratos_application source.